apps/expo-go/android/expoview/build.gradle

```diff
@@ -124,7 +130,14 @@ dependencies {
   api fileTree(dir: 'libs', include: ['*.jar'])
   implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlinVersion"
   implementation "org.jetbrains.kotlin:kotlin-reflect:$kotlinVersion"
-  implementation "androidx.lifecycle:lifecycle-process:2.8.7"
+  implementation 'androidx.lifecycle:lifecycle-process:2.8.7'
+  implementation 'androidx.compose.foundation:foundation-android:1.7.6'
+
+  implementation 'com.github.bumptech.glide:compose:1.0.0-beta01'
+  implementation 'io.coil-kt.coil3:coil-network-okhttp:3.1.0'
```

Comment:
post review: maybe `io.coil-kt.coil3:coil-network-okhttp:3.1.0` is unnecessary. 
thanks again for using glide